tests/Sabberworm/CSS/ParserTest.php

```diff
@@ -420,6 +420,12 @@ function testEmptyGridLineNameLenientInFile() {
 		$this->assertSame($sExpected, $oDoc->render());
 	}
 
+	function testUnmachedBracesInFile() {
+		$oDoc = $this->parsedStructureForFile('unmached_braces', Settings::create()->withMultibyteSupport(true));
```

Comment:
Typo in file name `unmatched_braces`…

Reply:
Lolz, not only in the file name :sweat_smile: